src/ast/helpers/key_value_options.rs

```diff
@@ -41,6 +39,13 @@ pub struct KeyValueOptions {
     pub delimiter: KeyValueOptionsDelimiter,
 }
 
+impl KeyValueOptions {
+    /// Returns true iff the options list is empty
+    pub fn is_empty(&self) -> bool {
+        self.options.is_empty()
+    }
+}
+
```

Comment:
I'm thinking we could skip the added API and have the caller call `value.options.is_empty()` instead? since this is only a thin wrapper might not be worth it